wsapi.go

```diff
@@ -426,12 +428,22 @@ type requestGuildMembersOp struct {
 // query     : String that username starts with, leave empty to return all members
 // limit     : Max number of items to return, or 0 to request all members matched
 // presences : Whether to request presences of guild members
-func (s *Session) RequestGuildMembers(guildID string, query string, limit int, presences bool) (err error) {
+// userIDs   : Used to specify which users you wish to fetch
+// nonce     : Nonce to identify the Guild Members Chunk response
+func (s *Session) RequestGuildMembers(guildID, query string, limit int, presences bool, userIDs *[]string, nonce string) (err error) {
```

Comment:
Otherwise, if we're keeping current approach, I think you can remove pointer from `userIDs` and do an internal check for length (`len(userIDs) > 0`).